tutorials/1. Basic PDE solving.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<p>First sentence: <code>PyDEns</code>-model -&gt; <code>PyDEns</code>-framework</p><p> </p><p>Add also a linking-sentence in the end of the paragraph. Something like</p><p> </p><p>"Let's start with the general setup of the PDE-problems solved by <code>PyDEns</code>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/pydens/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>